### PR TITLE
Update Serial.md

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]

**Review by ctzsnooze — DISMISSED** [review]

File: docs/development/Serial.md [hunk 1]
```diff
@@ -93,11 +93,24 @@ Note: for Identifier see serialPortIdentifier_e in the source; for Function bitm
...
+- ID's 40-49 reserved for LPUART 1-10
+- Other devices can be added starting from id 50.
+
+:::
+
+:::info
+
+In firmware 4.5 for SOFTSERIAL or LPUART we now for example have the following resources:
+
+resource SOFTSERIAL_TX 1 <PIN>
```
Comment (nerdCopter) [inline docs/development/Serial.md:110]:
```suggestion
`resource SOFTSERIAL_TX 1 <PIN>`
```

File: docs/development/Serial.md [hunk 2]
```diff
@@ -93,11 +93,24 @@ Note: for Identifier see serialPortIdentifier_e in the source; for Function bitm
...
+resource LPUART_TX 1 <PIN>
```
Comment (nerdCopter) [inline docs/development/Serial.md:111]:
```suggestion
`resource LPUART_TX 1 <PIN>`
```

**Comment by nerdCopter** [comment]
image

will test my recommends to see if this is resolved.

File: docs/development/Serial.md (hunk 2 above)
Comment (nerdCopter) [reply docs/development/Serial.md:111]: resolves compile, but incorrect. i guess 3 backticks around the pair is more correct.
image
--->
image

**Comment by haslinghuis** [comment]
We have to decide how to deal with LPUART. Now we use SERIAL_[T|R]X 11 to configure LPUART. But we have 20 serial ports reserved. I had prepared a PR to fix it but it was rejected. Now when a manufacturer want to add serial port 11, we have to configure LPUART with 12 instead - I think.

**Review by nerdCopter — APPROVED** [review]

**Comment by nerdCopter** [comment]
current render:
image

**Review by nerdCopter — APPROVED** [review]

**Comment by haslinghuis** [comment]
Now we just use UART 11 to configure LPUART. So the range for UARTs is wrong as it uses 0-19.

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**